src/components/calcite-accordion/calcite-accordion.e2e.ts

```diff
@@ -130,6 +131,26 @@ describe("calcite-accordion", () => {
     expect(await item3Content.isVisible()).toBe(true);
   });
 
+  it("clicking on the nested child accordion-item does not toggle the parent in single selection mode", async () => {
+    const page = await newE2EPage({
+      html: html`<calcite-accordion selection-mode="single" id="first"> ${accordionContent} </calcite-accordion>
+        <calcite-accordion selection-mode="single" id="second"> ${accordionContent} </calcite-accordion>`
+    });
```

Comment:
For test stability, can you add a `await page.waitForChanges()` after creating the page? By contrast, when you call `page.setContent()` it calls `waitForChanges` internally.

Reply:
Thanks for these tips, these are great!